src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvOrganizationEntityUpdater.java

```diff
@@ -108,11 +115,23 @@ private void updateOrganization(OrganizationId organizationId) {
     if (response.isOk()) {
       Identifier identifier = identifierController.findIdentifierById(organizationId);
       if (identifier == null) {
-        identifierController.createIdentifier(organizationId);
+        identifier = identifierController.createIdentifier(organizationId);
       }
+      
+      index(identifier.getKuntaApiId(), response.getResponse());
     } else {
       logger.warning(String.format("Organization %s processing failed on [%d] %s", organizationId.getId(), response.getStatus(), response.getMessage()));
     }
   }
 
+  private void index(String organizationId, Organization organization) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "index" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)